incubator/docker-registry/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+version: 0.1.0
```

Comment:
`appVersion` should reflect what it means as opposed to the chart version. In your case it should be `2`.